### DISCUSSION TITLE
Docker latest stale

### DISCUSSION THREAD

**Discussion by pepa65** [bpatrik/pigallery2#955] [Q&A] [answered]
I've been running the `bpatrik/pigallery2:latest` docker image with no issues. But I just looked at the dockerhub page and noticed that other docker images are updated while latest does not. Can I just switch over to for example `edge-alpine`?

**Comment by skatsubo** [comment]
`edge-alpine` is the right choice for sure.
It includes such neat fixes/features as
- https://github.com/bpatrik/pigallery2/pull/827
- https://github.com/bpatrik/pigallery2/pull/841
- https://github.com/bpatrik/pigallery2/pull/859

**Reply by pepa65** [reply]
Can I just switch to it (keeping the backend) by replacing `bpatrik/pigallery2:latest` with `bpatrik/pigallery2:edge-alpine`?

**Reply by skatsubo** [reply]
Yes, I think so.
I switched from `2.0.0` to `edge` this way by replacing the image.

**Comment by pepa65** [comment, marked as answer]
OK, did `docker-compose down`, then changed the image in `docker-compose.yml` to `pigallery2:edge-alpine`, then did `docker-compose up -d`. It came up, but I ran into Issue #961 which I fixed according to the last comment on that Issue. All good!

**Reply by pepa65** [reply]
Followup: today I could no longer use `docker-compose` due to a "ContainerError". I had to install package `docker-compose-v2` (I'm on Ubuntu) in order to be able to use `docker compose` (note the space). Then the container started, except somehow all users were erased... But admin:admin worked, and I added users back.
